db/migrate/20200909035016_change_user_pitchto_user_voice_name.rb

```diff
@@ -0,0 +1,4 @@
+class ChangeUserPitchtoUserVoiceName < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
I accidentally migrated this empty, that is why there are two migrations with similar names.

Reply:
can we take one of the migrations out?